### PR TITLE
Release 5.3.0

### PR DESCRIPTION
**Pull request by jesconstantine**
# Mayflower Release Notes

## 5.3.0
We've added some functionality to the location listings organism ([@organisms/by-author/location-listing](http://mayflower.digital.mass.gov/?p=organisms-location-listing))!  You can sort, filter, and paginate listing results! 

Note: The location listing organism is made up of lots of other patterns:
  - [@molecules/location-filters](http://mayflower.digital.mass.gov/?p=molecules-location-filters)
  - [@molecules/results-heading](http://mayflower.digital.mass.gov/?p=molecules-results-heading)
  - [@molecules/google-map](http://mayflower.digital.mass.gov/?p=molecules-google-map)
  - [@molecules/pagination](http://mayflower.digital.mass.gov/?p=molecules-pagination)
  - [@organisms/by-author/image-promos](http://mayflower.digital.mass.gov/?p=organisms-image-promos)
  
  And of course, you can still use those patterns in other components!

#### Confirm component works in context of page:

- [ ] Browse to http://stage-mayflower.digital.mass.gov/?p=pages-location-listing
- [ ] Confirm the zipcode filter is functional
  - [ ] Has an autocomplete with google places api
  - [ ] Results will *sort* based on proximity and update map accordingly
- [ ] Confirm the checkbox (tag) filters are functional
  - [ ] Results will *filter* based on existing tag in listing (verify by matching icons in listing/checkbox)
- [ ] Confirm pagination is functional
  - [ ] Can navigate between pages and see listing, map, results heading update
- [ ] Verify results heading is functional
  - [ ] Number of results per page / total results is dynamic
  - [ ] Matching tags per form input appear
  - [ ] Clicking on visible tag links removes the criteria and component updates accordingly

Note: Out of scope for this release (the keyword and activity filter)

#### Confirm component code didn't break child components on existing pages:

- [ ] Verify maps still render and are functional elsewhere
  - [ ] See banner on: http://stage-mayflower.digital.mass.gov/?p=pages-location-park-content 
  - [ ] Click marker
- [ ] Verify image promos still render
  - [ ] See "Activities" on: http://stage-mayflower.digital.mass.gov/?p=pages-location-park-content 

#### Confirm component code doesn't execute on child components in other contexts
- [ ] Verify results headings + tag interaction code doesn't execute (and that interaction is error free in console) on component page: http://stage-mayflower.digital.mass.gov/?p=molecules-results-heading
- [ ] Verify pagination code doesn't execute (and that interaction is error free in console) on component page: http://stage-mayflower.digital.mass.gov/?p=molecules-pagination

### CODE REVIEW

File: styleguide/source/_patterns/02-molecules/results-heading.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
     "tags": [{
       "text": "Item"
     },{
-      "text": "Item"
+      "type": "item-type",
+      "text": "Item",
+      "value": "item-value"
```
Comment (gender404) [inline styleguide/source/_patterns/02-molecules/results-heading.json]: The other tags should have these variables as well.  I also don't see a corresponding .md file update.

**Review by isaacchansky — APPROVED** [review]

File: styleguide/source/_patterns/02-molecules/results-heading.json (hunk 1 above)
Comment (jesconstantine) [reply styleguide/source/_patterns/02-molecules/results-heading.json]: Thanks @legostud update the data and created the pattern `.md` file in https://github.com/massgov/mayflower/pull/484/commits/80584ecfe9535730158078220933d52bd57b758e

**Comment by jesconstantine** [comment]
@legostud @isaacchansky Thanks for your reviews here.  I've updated the pagination and the documentation.  See: http://stage-mayflower.digital.mass.gov/?p=pages-location-listing

**Review by gleroux02 — APPROVED** [review]
👍 ran through each test, and all passed. Looks good to me.

**Pull request merged**